packages/lab/src/stepped-tracker/index.ts

```diff
@@ -1,3 +1,17 @@
+import type { Step } from "./Step";
+import type { SteppedTracker } from "./SteppedTracker";
+import type { StepReducer } from "./stepReducer";
+
 export * from "./SteppedTracker";
-export * from "./TrackerStep";
-export * from "./StepLabel";
+export interface SteppedTrackerProps extends SteppedTracker.Props {}
+export type SteppedTrackerOrientation = SteppedTracker.Orientation;
+
+export * from "./Step";
+export interface StepProps extends Step.Props {}
+export type StepStatus = Step.Status;
+export type StepStage = Step.Stage;
+export type StepDepth = Step.Depth;
+
+export * from "./useStepReducer";
+export type SteppedReducerState = StepReducer.State;
```

Comment:
why not just name them the same way as we export, so they match when a repo is searched ?